apps/build-onchain-apps/src/onchain/contract/OZ1155.ts

```diff
@@ -0,0 +1,479 @@
+const abi = [
+  {
```

Comment:
Great question @Sneh1999, we can subset the abi to just the functions we use.

Pros:
- saves some network transfer
- can show abi at point of use in the code instead of importing

Cons:
- maybe cumbersome if we want to do more contract interaction in the future

This contract is so basic that I doubt the network savings are much (few KB, less when compressed), but by the same token we're already using all the interesting functions on this contract so it wouldn't hurt us to subset. I don't have a strong opinion on this one, do you have a preference?

Reply:
The reason I am suggesting this change is because we are creating this project as a baseline of good practices/products. 

It might be that in our use case doing this might not be that helpful but for someone else the abi might be pretty big and storing the whole api then would not be a good practice. 

@cnasc

Reply:
That's a fair point, I'll extract the functions we use.